react/utils/categoryMenuDisposition.js

```diff
@@ -0,0 +1,26 @@
+import { values, map } from 'ramda'
+
+const categoryMenuDisposition = {
+  DISPLAY_CENTER: {
+    name: 'editor.category-menu.disposition-type.center',
+    value: 'center',
+  },
+  DISPLAY_LEFT: {
+    name: 'editor.category-menu.disposition-type.left',
+    value: 'left',
+  },
+  DISPLAY_RIGHT: {
+    name: 'editor.category-menu.disposition-type.right',
+    value: 'right',
+  },
+}
+
+export function getMenuDispositionNames() {
+  return map(opt => opt.name, values(categoryMenuDisposition))
```

Comment:
Yeap. And also, just as a sidenote, instead of `opt => opt.name` you could've used `prop('name')`. Samesis to next function

Everytime you are using two ramda functions to access some property or generate a data structure, be certain that there are none other function that does that already, because it's very probable

Reply:
I was replying to @GustavoMA here